setup.py

```diff
@@ -24,9 +24,9 @@ def get_version(filename):
     include_package_data=True,
     install_requires=[
         'setuptools',
-        'requests >= 2.2.1',
-        'pafy >= 0.3.35',
-        'Mopidy >= 1.0',
+        'requests >= 2.7.0',
```

Comment:
This will not work with python-requests from e.g. Ubuntu 15.04, which has requests 2.4.3. Isn't requests quite stable in the entire 2.x series?